server/service/handler.go

```diff
@@ -355,6 +355,8 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 
 	ue.GET("/api/_version_/fleet/activities", listActivitiesEndpoint, listActivitiesRequest{})
 
+	ue.GET("/api/_version_/fleet/download_installer/{enroll_secret}/{kind}", getInstallerEndpoint, installerRequest{})
```

Comment:
@lucasmrod should we rate-limit this endpoint? should we rate-limit _all_ sandbox endpoints by default?